StreamingAnalytics/LowLatencySample/Temperature-Epl/monitors/TemperatureEventMonitor.mon

```diff
@@ -0,0 +1,29 @@
+package lowlatency;
```

Comment:
Don't really need package name. I don't think the other examples have it.

Reply:
Done